python/paddle/fluid/dygraph/dygraph_to_static/logging_utils.py

```diff
@@ -0,0 +1,225 @@
+            rv = level
+        elif str(level) == level:
+            if level not in name_to_level_dict:
+                raise ValueError("Unknown level: %r" % level)
+            rv = name_to_level_dict[level]
+        else:
+            raise TypeError("Level not an integer or a valid string: %r" %
+                            level)
+        return rv
+
+    def has_code_level(self, level):
+        level = self.check_level(level, _transformer_name_to_level)
+        return level == self.transformed_code_level
+
+    def has_verbosity(self, level):
+        level = self.check_level(level, {})
+        return level >= self.verbosity_level
+
+    def error(self, msg, *args, **kwargs):
+        self.logger.error(msg, *args, **kwargs)
+
+    def warn(self, msg, *args, **kwargs):
+        self.logger.warn(msg, *args, **kwargs)
+
+    def log(self, level, msg, *args, **kwargs):
+        if self.has_verbosity(level):
+            self.logger.log(level, msg, *args, **kwargs)
+
+    def log_transformed_code(self, level, ast_node, *args, **kwargs):
+        if self.has_code_level(level):
+            msg = ast_to_source_code(ast_node)
+            msg = "Transformed code: \n" + msg
+            self.logger.info(msg, *args, **kwargs)
+
+
+_TRANSLATOR_LOGGER = TranslatorLogger()
+
+BasicApiTransformer = 1
+TensorShapeTransformer = 2
+ListTransformer = 3
+BreakContinueTransformer = 4
+ReturnTransformer = 5
+LogicalTransformer = 6
+LoopTransformer = 7
+IfElseTransformer = 8
+AssertTransformer = 9
+PrintTransformer = 10
+CallTransformer = 11
+CastTransformer = 12
+AllTransformer = 12
+
+_transformer_name_to_level = {
+    'BasicApiTransformer': BasicApiTransformer,
+    'TensorShapeTransformer': TensorShapeTransformer,
+    'ListTransformer': ListTransformer,
+    'BreakContinueTransformer': BreakContinueTransformer,
+    'ReturnTransformer': ReturnTransformer,
+    'LogicalTransformer': LogicalTransformer,
+    'LoopTransformer': LoopTransformer,
+    'IfElseTransformer': IfElseTransformer,
+    'AssertTransformer': AssertTransformer,
+    'PrintTransformer': PrintTransformer,
+    'CallTransformer': CallTransformer,
+    'CastTransformer': CastTransformer,
+    'AllTransformer': AllTransformer
+}
+
+
+def set_verbosity(level=0):
+    """
+    Sets the verbosity level for dygraph to static graph.
+
+    There are two means to set the logging verbosity:
+     1. Call function `set_verbosity`
+     2. Set environment variable `TRANSLATOR_VERBOSITY`
+    NOTE: `set_verbosity` has higher priority than the environment variable
+
+    Args:
+        level(int): The verbosity level. The larger value idicates more verbosity.
+            The default value is 0, which means no logging.
+    Examples:
+        .. code-block:: python
+
+            import os
+            import paddle.fluid as fluid
+
+            fluid.dygraph.dygraph_to_static.set_verbosity(1)
+            # The verbosity level is now 1
```

Comment:
Consider to expose interface as `paddle.jit.set_verbosity()`?

Reply:
done